tests/e2e/specs/out-of-game/stats.spec.js

```diff
@@ -253,6 +253,15 @@ describe('Stats Page', () => {
     });
   });
 
+});
+
+describe('Usage stats', () => {
+  beforeEach(setup);
+
+  it('Displays the site usage stats in the stats usage page', () => {
+    cy.get('#usage-stats-section').find('#usage-stats-chart');
+  });
```

Comment:
@seriouslysean per our convo, I've grouped the existing test on the data sent from the backend with this new one that just asserts that the chart appears